rubix-tests/src/test/java/com/qubole/rubix/tests/TestThriftServerJVM.java

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2018. Qubole Inc
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. See accompanying LICENSE file.
+ */
+package com.qubole.rubix.tests;
+
+import com.qubole.rubix.core.utils.DataGen;
+import com.qubole.rubix.core.utils.DeleteFileVisitor;
+import com.qubole.rubix.spi.BlockLocation;
+import com.qubole.rubix.spi.BookKeeperFactory;
+import com.qubole.rubix.spi.CacheConfig;
+import com.qubole.rubix.spi.CacheUtil;
+import com.qubole.rubix.spi.Location;
+import com.qubole.rubix.spi.RetryingBookkeeperClient;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.conf.Configured;
+import org.apache.hadoop.fs.Path;
+import org.apache.thrift.shaded.TException;
+import org.apache.thrift.shaded.transport.TTransportException;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by kvankayala on 10 Jul 2018.
+ * TestThriftServerJVM Class created to test separate JVMs for bookKeeper and LDS servers
+ */
+public class TestThriftServerJVM extends Configured
+{
+  private static final String testDirectoryPrefix = System.getProperty("java.io.tmpdir") + "/TestThriftServerJVM/";
```

Comment:
As we discussed, this issue will not be handled in this PR because the changes that jordan had made were in rubix-bookkeeper module which are currently not available in rubix-tests module